pdr_backend/accuracy/app.py

```diff
@@ -285,61 +307,43 @@ def save_statistics_to_file():
         "0x4ac2e51f9b1b0ca9e000dfe6032b24639b172703", network_param
     )
 
-    contracts_list_unfiltered = fetch_contract_id_and_spe(
+    contracts_list_unfiltered: List[ContractIdAndSPE] = fetch_contract_id_and_spe(
         contract_addresses,
         network_param,
     )
+    output = []
```

Comment:
please do not use `output` as the name, since it indicates some sort of permanence/outputting to the screen or a file. I suggest `result` instead.